R/modeltests.R

```diff
@@ -304,17 +304,7 @@ evaluateRuns <- function(model, mydir, gitPath, compScen, email, mattermostToken
         system(paste("rsync -e ssh -av fulldata.gdx", gdxOnRseServer))
       }
       cfg <- NULL
-      # Question: is this check really useful? 'Conv' has been checked 4 lines above already.
-      # Doesn't this check exclude runs that have 'converged (had INFES)' and
-      # wouldn't it be useful to also produce compareScenarios for these? The existence of the mif file only
-      # matters later on after the runtime check when it comes to compareScenarios. So why testing for it here?
-      if (any(grepl(basename(getwd()), rownames(filter(gRS, .data$Conv == "converged", .data$Mif %in% c("yes", "sumErr")))))) {
-        load("config.Rdata")
-      } else {
-        setwd("../")
-        message("Skipping ", i, " and changed back to ", normalizePath("."))
-        next
-      }
+      load("config.Rdata")
       sameRuns <- gRS %>% filter(.data$Conv %in% c("converged", "converged (had INFES)"), # runs have to be converged
```

Comment:
I think a cs2 is useful even if the run did not converge, specifically because one would want to look at why it didn't.

Reply:
I'm not sure which runs you are interested in. I assume you mean the ones that are feasible but didn't make it to 100 iterations (status "not_converged"), right? Because runs that are infeasible don't write a mif file, so no cs2 could be run.

Reply:
Exactly, those with status `not_converged`

Reply:
Done